Test/Microsoft.ApplicationInsights.Test/Shared/TelemetryClientExtensionTests.cs

```diff
@@ -282,12 +282,37 @@ public void StartOperationWithOperationTelemetrySetsOperationHolderTelemetry()
             Assert.AreEqual(requestTelemetry, this.sendItems[0]);
         }
 
+        [TestMethod]
+        public void StopOperationWhenTelemetryIdDoesNotMatchActivityId()
```

Comment:
May it break manual telemetry correlation that customer implemented on their own? We had multiple users doing manual setup of IDs/ParentIDs before. With us matching IDs to Activity, it looks like we may drop something user intended to track with custom IDs. Is it the case here or I'm overthinking this? :)

Reply:
it was the behavior we had for years. nothing has changed with this pr, except I added test that verifies it ;)

Also, while users play with operationids, and parents, this only checks for telemtery id.